prisma/seeds/prices.ts

```diff
@@ -14,6 +14,45 @@ interface PriceFileData extends KeyValueT<string> {
   priceInUSD: string
 }
 
+export const mockPrices: Price[] = [
+  { // XECUSD
+    id: 0,
```

Comment:
All of them having the same if shouldn't result in an error? I see in the test that it passed fine, just trying to understand.

Reply:
since the ID is defined as:
`id           Int                    @id @default(autoincrement())`

I believe it uses autoincrement instead of the value (zero) passed in there, effectively ignoring it (that is an educated guess, this doubt also hit me earlier). Since it is a required field, I had to pass a number, zero seemed the best one for clarity. I also input zero to all of them when reading the file and inserting thousands into the database.